_posts/2019/2019-03-24-protect-your-sign-up-form-using-rack-attack.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Protect your sign up form with Rack::Attack"
+description: "Slow down the malicious robots"
+layout: article
+category: ruby
+image:
+  base: "2019/protect-your-sign-up-form-using-rack-attack"
+  alt: "Clipboard sign up"
+  source: "https://unsplash.com/photos/lUShu7PHIGA"
+  credit: "Eco Warrior Princess"
+---
+
+When your application becomes popular, in may attract the attention of hackers, who’ll try and find ways to exploit the weaknesses in your site to use it for nefarious means!
+
+They’ll nearly always explore your site manually, signing up and testing attack vectors, before attempting to automate the weaknesses they’ve established.
```

Comment:
*they've discovered (right? Because you're the person who 'establishes' the weakness, in a way...)